bedrock/settings/base.py

```diff
@@ -258,6 +258,59 @@ def lazy_langs():
     'csp-violation-capture',
 ]
 
+# Pages that we don't want be indexed by search engines.
+NOINDEX_URLS = [
+    r'^(404|500)/',
+    r'^about/partnerships/contact-bizdev/',
+    r'^contribute/(embed|event)/',
+    r'^csp-violation-capture',
+    r'^firefox/sms/sent/',
+    r'^firefox/unsupported/',
+    r'^firefox/send-to-device-post',
+    r'^firefox/feedback',
+    r'^firefox/stub_attribution_code/',
+    r'^.+/(firstrun|whatsnew)/$',
+    r'^infobar/',
+    r'^l10n_example/',
+    r'^m/',
+    r'^newsletter/(confirm|existing|hacks\.mozilla\.org|recovery|updated)/',
+    r'^tabzilla/',
+    r'/system-requirements/$',
+    r'.*/(firstrun|thanks)/$',
+    r'^rna/',
+    r'^healthz/$',
+    r'^country-code\.json$',
+    # exclude redirects
+    r'^foundation/annualreport/$'
+    r'^firefox/notes/$'
+    r'^teach/$'
+    r'^thunderbird/(release)?notes/$'
+    r'^about/legal/impressum/$',
+    r'^security/announce/'
+]
+
+# Pages we do want indexed but don't show up in automated URL discovery
+# or are only available in a non-default locale
+EXTRA_INDEX_URLS = [
```

Comment:
@hoosteeno Just added these manually as they redirect in "en-US", but not in "de".